### PR TITLE
Create ui-c8y-1018-503-114-block-searching-when-creating-editing-private-smartrule-graft-release-y-2024.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by pmic

Ticket: [MTM-55913](https://cumulocity.atlassian.net/browse/MTM-55913)
Original PR: [6819](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6819)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1018-503-114-block-searching-when-creating-editing-private-smartrule-graft-release-y-2024.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: block searching when creating editing private smartrule graft release y2024 (#6819)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-114-block-searching-when-creating-editing-private-smartrule-graft-release-y-2024.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Block searching when creating or editing private smart rules
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-114-block-searching-when-creating-editing-private-smartrule-graft-release-y-2024.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-55913
+version: 1018.503.114
+---
+block searching when creating editing private smartrule graft release y2024 (#6819)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-114-block-searching-when-creating-editing-private-smartrule-graft-release-y-2024.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the past, it was possible to search for devices when creating or editing private smart rules, which could potentially lead to incorrect device assignments. With this change, searching for devices is now blocked when creating or editing private smart rules. This ensures that only explicitly selected devices are assigned to the smart rule, preventing unintended device assignments and improving the accuracy and reliability of private smart rules in the system.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1018-503-114-block-searching-when-creating-editing-private-smartrule-graft-release-y-2024.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Block searching when creating or editing private smart rules
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-114-block-searching-when-creating-editing-private-smartrule-graft-release-y-2024.md:3]:
```suggestion
title: Block device search when creating or editing private smart rules
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-114-block-searching-when-creating-editing-private-smartrule-graft-release-y-2024.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-55913
+version: 1018.503.114
+---
+In the past, it was possible to search for devices when creating or editing private smart rules, which could potentially lead to incorrect device assignments. With this change, searching for devices is now blocked when creating or editing private smart rules. This ensures that only explicitly selected devices are assigned to the smart rule, preventing unintended device assignments and improving the accuracy and reliability of private smart rules in the system.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-114-block-searching-when-creating-editing-private-smartrule-graft-release-y-2024.md:17]:
```suggestion
In the past, it was possible to search for devices when creating or editing private smart rules, which could potentially lead to incorrect device assignments. With this change, searching for devices is now blocked when creating or editing private smart rules. This ensures that only explicitly selected devices are assigned to the smart rule, preventing unintended device assignments and improving the accuracy and reliability of private smart rules in the platform.
```

**Comment by BeateRixen** [comment]
@rahultsag Could you please check and approve as Jan is on vacation.

**Review by darluk — APPROVED** [review]

**Review by rahultsag — APPROVED** [review]

**Pull request merged**